### ISSUE TITLE
Is input of tf.contrib.cudnn_rnn.CudnnLSTM time-major?

### ISSUE THREAD

**Issue by scotthuang1989** [tensorflow/tensorflow#18318]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: 17.10
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.6
- **Python version**: 3.6
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:9/7.0
- **GPU model and memory**:
- **Exact command to reproduce**:

### Describe the problem

I save this in tensorflow/models: [quick_draw exmple](https://github.com/tensorflow/models/blob/master/tutorials/rnn/quickdraw/train_model.py)

you can see that in the comment of this function.  it says that CudnnLSTM is time-major, so it need a transpose. But I don't find this information in documentation [here](https://tensorflow.google.cn/versions/r1.6/api_docs/python/tf/contrib/cudnn_rnn/CudnnLSTM#__call__):

Where can I get confirmation of this ?
```
  def _add_cudnn_rnn_layers(convolved):
    """Adds CUDNN LSTM layers."""
    # Convolutions output [B, L, Ch], while CudnnLSTM is time-major.
    convolved = tf.transpose(convolved, [1, 0, 2])
    lstm = tf.contrib.cudnn_rnn.CudnnLSTM(
        num_layers=params.num_layers,
        num_units=params.num_nodes,
        dropout=params.dropout if mode == tf.estimator.ModeKeys.TRAIN else 0.0,
        direction="bidirectional")
    outputs, _ = lstm(convolved)
    # Convert back from time-major outputs to batch-major outputs.
    outputs = tf.transpose(outputs, [1, 0, 2])
    return outputs

```

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 16 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by xiaodongww** [comment]
I also meet this problem and it takes hours for me to figure out that CudnnLSTM is time-major according to this [answer](https://stackoverflow.com/questions/49183538/simple-example-of-cudnngru-based-rnn-implementation-in-tensorflow).    
For a beginner like me, it is also likely to use CudnnLSTM as a substitution of RNNCell and falsely use it as the input of tf.nn.bidirectional_dynamic_rnn.   
Would it be possible to clarify the input and output of tf.contrib.cudnn_rnn.CudnnLSTM more clearly in the [documentation](https://www.tensorflow.org/api_docs/python/tf/contrib/cudnn_rnn/CudnnLSTM)?  
Thanks!
